package.json

```diff
@@ -27,17 +27,7 @@
   "devDependencies": {
     "@svgr/cli": "^8.1.0",
     "@svgr/webpack": "^8.1.0",
-    "gh-pages": "^5.0.0",
-    "husky": "^8.0.3",
-    "lint-staged": "^14.0.1",
-    "prettier": "^3.0.3",
-    "prettier-plugin-tailwindcss": "^0.5.3",
-    "source-map-loader": "^4.0.1",
-    "tailwindcss": "^3.3.3"
-  },
-  "devDependencies": {
-    "@svgr/cli": "^8.1.0",
-    "@svgr/webpack": "^8.1.0",
+    "eslint": "^8.50.0",
```

Comment:
The only actual change was adding/installing `eslint` here, but there were two `devDependencies` in `master` that I had to resolve.